tests/integration_tests.rs

```diff
@@ -15,11 +15,6 @@
 // along with this program. If not, see
 // <https://www.gnu.org/licenses/>.
 
-extern crate assert_cmd;
-extern crate assert_fs;
-extern crate predicates;
-extern crate rand;
-
```

Comment:
100% glad to see this go. I thought I had removed them already, but apparently haven't.